django_mongodb/fields/object_id.py

```diff
@@ -0,0 +1,37 @@
+from bson import ObjectId
+from bson.errors import InvalidId
+from django.core import exceptions
+from django.db.models.fields import Field
+from django.utils.translation import gettext_lazy as _
+
+
+class ObjectIdField(Field):
+    default_error_messages = {
+        "invalid": _("“%(value)s” value must be an Object Id."),
+    }
+    description = _("ObjectId")
+
+    def get_internal_type(self):
+        return "ObjectIdField"
+
+    def db_type(self, connection):
+        return "objectId"
+
+    def rel_db_type(self, connection):
+        return "objectId"
+
+    def to_python(self, value):
+        try:
+            return ObjectId(value)
+        except (TypeError, InvalidId):
+            raise exceptions.ValidationError(
+                self.error_messages["invalid"],
+                code="invalid",
+                params={"value": value},
+            ) from None
+
+    def deconstruct(self):
+        name, path, args, kwargs = super().deconstruct()
+        if path.startswith("django_mongodb.fields.auto"):
```

Comment:
auto -> objectid (I think we should omit the underscore from the file name to match `bson/objectid.py`)

Reply:
shall I rename auto to objectid.py and put all together? or only rename object_id.py ?